commons/c2cgeoportal_commons/lib/email_.py

```diff
@@ -81,10 +81,12 @@ def send_email(
         smtp: smtplib.SMTP = smtplib.SMTP_SSL(smtp_config["host"])
     else:
         smtp = smtplib.SMTP(smtp_config["host"])
-    if smtp_config.get("user", False):
-        smtp.login(smtp_config["user"], smtp_config["password"])
     if smtp_config.get("starttls", False):
         smtp.starttls()
+    if smtp_config.get("user", False):
+        smtp.login(smtp_config["user"], smtp_config["password"])
+    if smtp_config.get("user", False):
+        smtp.login(smtp_config["user"], smtp_config["password"])
```

Comment:
it seems there is the same group of lines twice.

Reply:
Merge issue effectively, thanks :-)